cruise.umple/src/UmpleInternalParser_CodeMixset.ump

```diff
@@ -0,0 +1,178 @@
+/*
+
+Copyright: All contributers to the Umple Project
+
+This file is made available subject to the open source license found at:
+http://umple.org/license
+
+This file analyze core umple meta model tokens such as which language to generate,
+the current namespace, etc and populates the umple meta model.
+
+Please refer to UmpleInternalParser.ump for more details.
+
+*/
+
+
+namespace cruise.umple.compiler;
+
+class UmpleModel{
+  depend java.util.stream.*;
+
+/*
+ * returns a mixset from UmpleModel based on its name.
+ * @Abdulaziz   
+ */
+ Mixset getMixset(String name) {
+	Mixset mixset = null;
```

Comment:
Please use two spaces throughout Umple, never tabs. You may find tabs in some places but that is not our standard.

Reply:
OK.